ios/Classes/BetterPlayerPlugin.m

```diff
@@ -431,6 +470,15 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
             result(@([player absolutePosition]));
         } else if ([@"seekTo" isEqualToString:call.method]) {
             [player seekTo:[argsMap[@"location"] intValue]];
+
+            if(!_isCommandCenterButtonsEnabled){
```

Comment:
I feel strange using `_isCommandCenterButtonsEnabled` at `seekTo`, not `play`. 
Do ing this on `play` is not work?

Reply:
@HiroshiOshiro 

> I feel strange using `_isCommandCenterButtonsEnabled` at `seekTo`, not `play`.

I using it in both `seekTo` and `play` (`play` event will call `setupRemoteCommands`).

> Doing this on `play` is not work?

if only doing it on `play`, after playback ended, users can seek back in detail screen, then we should re-enable the `Play` button to make the users can play the video at that time (our current logic won't auto play after seeking from the end).

Reply:
I see, thanks